header.php

```diff
@@ -47,4 +47,42 @@
         </div>
       </div>
     </div>
-  </header>
\ No newline at end of file
+  </header>
+
+  <header id="mobile-header" class="header padding-top-tiny padding-bottom-tiny">
+    <div class="container">
+      <div class="grid-row align-items-center justify-between">
+        <div class="grid-item item-s-1">
+          <span href="#" id="menu-toggle">
+            <span class="open-menu">
+              <?php echo url_get_contents(get_template_directory_uri() . '/dist/img/menu.svg'); ?>
+            </span>
+            <span class="close-menu">
+              <?php echo url_get_contents(get_template_directory_uri() . '/dist/img/cancel.svg'); ?>
+            </span>
+
+</span>
+        </div>
+        <div class="grid-item item-s-1">
```

Comment:
this needs to be 10 cols wide and text aligned center

Reply:
or. lets put the toggle in s-4 left aligned, and the logo in s-4 centered, and then we have more space to set the widths of each visually